core/common/misp_to_yeti.py

```diff
@@ -0,0 +1,63 @@
+from core.schemas import observable
+MISP_TYPES_TO_IMPORT = {
+        "domain": observable.ObservableType.hostname,
+        "hostname": observable.ObservableType.hostname,
+        "ip-dst": observable.ObservableType.ipv4,
+        "ip-src": observable.ObservableType.ipv4,
+        "url": observable.ObservableType.url,
+        "md5": observable.ObservableType.md5,
+        "sha1": observable.ObservableType.sha1,
+        "sha256": observable.ObservableType.sha256,
+        "filename|sha256": observable.ObservableType.sha256,
+        "filename|md5": observable.ObservableType.md5,
+        "filename|sha1": observable.ObservableType.sha1,
+        "ssdeep": observable.ObservableType.ssdeep,
+        "mutex": observable.ObservableType.mutex,
+        "named pipe": observable.ObservableType.named_pipe,
+        "btc": observable.ObservableType.wallet,
+        "email": observable.ObservableType.email,
+        "filename": observable.ObservableType.file,
+        "regkey": observable.ObservableType.registry_key,
+    }
+
+class MispToYeti:
+    
+    def __init__(self, misp_event):
+        self.misp_event = misp_event
+
+    def attr_misp_to_yeti(self, attribute:dict) -> observable.Observable:
+        if attribute.get("type") in MISP_TYPES_TO_IMPORT:
+            obs_yeti = observable.TYPE_MAPPING[MISP_TYPES_TO_IMPORT[attribute.get("type")]](value=attribute.get("value")).save()
+            print(f"Attribute {attribute.get('value')} imported")
+            return obs_yeti
+
+    def add_context_by_misp(self,attribute_misp:dict,event:dict,obs_yeti:observable.Observable)-> dict:
+        context = {}
+        event_id = attribute_misp.get("event_id")
+        context["Org"] = event.get("Org")['name']
+        context['event_id'] = event_id
+        if attribute_misp.get("comment"):
+            context['comment'] = attribute_misp.get("comment")
+        
+        obs_yeti.add_context('misp',context)
+
+    def obs_misp_to_yeti(self, object_misp:dict):
+        objs_type = object_misp.get("type")
+        links = []
+        for attr in object_misp.get("Attribute"):
+            obs_yeti = self.attr_misp_to_yeti(attr)
+            links.append(obs_yeti)     
+        obs_yeti = links.pop()
```

Comment:
Why pop() here? Why is the first item of the loop the one that gets links to everything else?